src/test/kotlin/no/nav/syfo/testhelper/TestEnvironment.kt

```diff
@@ -21,8 +21,8 @@ fun testEnvironment(
     pdlUrl = pdlUrl,
     skjermedePersonerPipClientId = "dev-gcp.nom.skjermede-personer-pip",
     skjermedePersonerPipUrl = skjermedePersonerPipUrl,
-    syfotilgangskontrollClientId = "dev-fss.teamsykefravr.syfo-tilgangskontroll",
-    syfotilgangskontrollUrl = syfotilgangskontrollUrl,
+    istilgangskontrollClientId = "dev-fss.teamsykefravr.istilgangskontroll",
```

Comment:
Er jo bare test, men for ryddighetens skyld? 😁
```suggestion
    istilgangskontrollClientId = "dev-gcp.teamsykefravr.istilgangskontroll",
```

Reply:
Lært av @eirikdahlen 🎖️